src/interpreter/index.ts

```diff
@@ -81,6 +81,9 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
         try {
             this._environment = newEnv;
             return func(this);
+        } catch (err) {
+            console.error("Runtime error encountered in BRS implementation: ", err);
```

Comment:
Travis tests are currently failing and that's entirely my fault!  I forgot I'm throwing errors to handle `return` statements in functions, `exit for` and `exit while` to break out of loops, and… maybe one more thing?  Anyway, this should fix it:

```suggestion
            if (err.kind == null) {
                console.error("Runtime error encountered in BRS implementation: ", err);
            }
```

(I'm testing github's "suggested changes" feature so if this looks weird or doesn't work or something blame github 🤷‍♂️ https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change)